cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/MultivariateAnomalyDetection.scala

```diff
@@ -457,6 +490,62 @@ class FitMultivariateAnomaly(override val uid: String) extends Estimator[DetectM
 
   protected def prepareUrl: String = getUrl
 
+  override protected def queryForResult(key: Option[String],
+                                        client: CloseableHttpClient,
+                                        location: URI): Option[HTTPResponseData] = {
+    val get = new HttpGet()
+    get.setURI(location)
+    key.foreach(get.setHeader("Ocp-Apim-Subscription-Key", _))
+    get.setHeader("User-Agent", s"synapseml/${BuildInfo.version}${HeaderValues.PlatformInfo}")
+    val resp = convertAndClose(sendWithRetries(client, get, getBackoffs))
+    get.releaseConnection()
+    Some(resp)
+  }
+
+  override protected def handlingFunc(client: CloseableHttpClient,
```

Comment:
https://github.com/microsoft/SynapseML/pull/1457/files

This PR added resiliency needed for MVAD. Could you please ensure that all the network calls flow through this function so that we can be a unified force against flaky network issues? This might already be the case but double checking this will help me sleep at night

Reply:
If you're meaning make sure FitMultivariateAnomaly's network goes inside this override function to query for result then yes, I've debugged the tests to see it go through this function. Or is there anything else you're thinking of?

Reply:
Thank you!